code/controllers/subsystems/inactivity.dm

```diff
@@ -27,6 +27,12 @@ SUBSYSTEM_DEF(inactivity)
 
 					else if(issilicon(C.mob))
 						information = " while a silicon."
+						if(isAI(C.mob))
+							var/mob/living/silicon/ai/A = C.mob
+							empty_playable_ai_cores += new /obj/structure/AIcore/deactivated(A.loc)
+							global_announcer.autosay("[A] has been moved to intelligence storage.", "Artificial Intelligence Oversight")
+							A.clear_client()
+							information = " while an AI."
```

Comment:
shouldve commented before this got merged; is this not going to wind up printing "[playername] was kicked for being AFK while a silicon while an AI?"

Reply:
Sets it equal to, doesn't add to.